pkg/operatorrules/registry_test.go

```diff
@@ -121,52 +121,36 @@ var _ = Describe("OperatorRules", func() {
 			Expect(registeredAlerts).To(ConsistOf(alerts))
 		})
 
-		It("should replace alerts with the same name in the same RegisterAlerts call", func() {
+		It("should replace alerts with the same name and same expression in the same RegisterAlerts call", func() {
```

Comment:
same as above.

also a question, since there is 3 tests that used same alerts name and expr why not define it in one place and just use alerts var 3 times?

Reply:
the expressions change a bit
I think it would become less clear as they are in different files